src/content/developers/tutorials/Using-UseDApp-with-unity/index.md

```diff
@@ -0,0 +1,206 @@
+sidebar: true
+sourceUrl: https://github.com/lucasespinosa28/usedappwithUnity
+---
+## Introduction {#introduction}
+In this tutorial, you will learn how to use [UseDApp](https://usedapp.readthedocs.io/en/latest/) a React plugins for rapid Dapp development, 
+and how to interact with a Unity using react, you will get your account balance inside a Unity and send a transaction from unity.
+
+### You need have [Unity](https://unity.com/) installed 
+
+## Unity project {#unity-project}
+On the Unity will have a text to receive the account balance, and a button to send a transaction using UseDApp,
+to communicate from unity to [UseDApp](https://usedapp.readthedocs.io/en/latest/) we will need an extra code using ```jslib```.
+
+### 1\. Create a new unity project and configure to webgl {#new-unity-project}
+Create a new unity project, the name of the project and if the project is 2d or 3d makes no difference. 
+![Captura de Tela (80)](https://user-images.githubusercontent.com/52639395/119444261-4e3a6980-bd01-11eb-9946-e7b7653918b3.png)
+
+Configure the project to build to WebGL to use in the browser, go to ```File > build settings``` choose Webgl, and click on the switch platform.
+
+![Captura de Tela (78)](https://user-images.githubusercontent.com/52639395/119444075-0a476480-bd01-11eb-8b00-a29c13a0b468.png)
+
+After switch to webgl go to ```Edit > Project Settings > Player ```, in publish setting choose disable compression format
+
+![Captura de Tela (77)](https://user-images.githubusercontent.com/52639395/119443979-e2f09780-bd00-11eb-9eea-2bed9ca5596d.png)
+
+### 2\. Create the interface{#create-UI}
+Now create the UI to interact with unity
+  1. Create a ```Canvas```.
+  2. Create a ```Text``` with name **BalanceText** to store the balance.
+  3. Create a ```Button```to call method to send a transaction and a ```Input Field``` to store the address for transaction.
+![Captura de Tela (90)](https://user-images.githubusercontent.com/52639395/119541875-4dd3ba00-bd65-11eb-913a-ba320f0dfc55.png)
+
+### 3\. Write code to receive the balance {#unity-balance}
+In asset folder create a new cs file ```BalanceController.cs```, drag and drop this file into gameobject ```BalanceText```,open the code file and writer the code below.
+```csharp
+    void EtherBalanceToUnity(string amount)
+    {
+        GameObject.Find("BalanceText").GetComponent<Text>().text = $"Balance: {amoun} EHT";
+    }
+```
+The **EtherBalanceToUnity** will receive the account balance from react app.
+
+### 4\. Write code to send a transaction {#unity-send-transaction}
+In asset folder create a new folder with name **Plugins**, create a new file ```script.jslib``` inside this new folder, 
+[jslib](https://docs.unity3d.com/Manual/webgl-interactingwithbrowserscripting.html) script are similar  to javascript,
+this script build a bridge between your Unity project and the react app,**SendTranscationFromUnitya** is the function that will exist in both unity and react app,write the code below.
```

Comment:
double-check spacing... extra space after "similar", missing space before "**SendTranscationFromUnitya**" which also has a stray "a" that needs removal at the end. Also missing space before last comma ("app, write").

Doubled check capitalization too... "JavaScript", "Unity", "React" are all names